networking/hub-regionA.json

```diff
@@ -387,9 +387,464 @@
                 ]
             }
         },
+        {
+            "type": "Microsoft.Network/firewallPolicies",
+            "apiVersion": "2020-11-01",
+            "name": "[variables('hubFwName')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/ipGroups', variables('aksIpGroupName'))]"
+            ],
+            "properties": {
+                "sku": {
+                    "tier": "Standard"
+                },
+                "threatIntelMode": "Deny",
+                "threatIntelWhitelist": {
+                    "ipAddresses": []
+                },
+                "dnsSettings": {
+                    "servers": [],
+                    "enableProxy": true
+                }
+            },
+            "resources": [
+                {
+                    "type": "Microsoft.Network/firewallPolicies/ruleCollectionGroups",
+                    "apiVersion": "2020-11-01",
+                    "name": "[concat(variables('hubFwName'), '/DefaultDnatRuleCollectionGroup')]",
+                    "location": "[parameters('location')]",
+                    "dependsOn": [
+                        "[variables('hubFwName')]"
+                    ],
+                    "properties": {
+                        "priority": 100,
+                        "ruleCollections": []
+                    }
+                },
+                {
+                    "type": "Microsoft.Network/firewallPolicies/ruleCollectionGroups",
+                    "apiVersion": "2020-11-01",
+                    "name": "[concat(variables('hubFwName'), '/DefaultApplicationRuleCollectionGroup')]",
+                    "location":"[parameters('location')]",
+                    "dependsOn": [
+                        "[variables('hubFwName')]",
```

Comment:
Is this supposed to be the policy, if so, can we reference it more specifically.

Reply:
let me try it out. I will let you know.

Reply:
done